scripts/install/deluge.sh

```diff
@@ -185,17 +196,17 @@ cat > /home/${u}/.config/deluge/hostlist.conf${SUFFIX} <<DHL
   ]
 }
 DHL
-
   echo "${u}:${pass}:10" > /home/${u}/.config/deluge/auth
   echo "localclient:${localpass}:10" >> /home/${u}/.config/deluge/auth
   chmod 600 /home/${u}/.config/deluge/auth
   chown -R ${u}.${u} /home/${u}/.config/
-  mkdir /home/${u}/dwatch
-  chown ${u}: /home/${u}/dwatch
-  mkdir -p /home/${u}/torrents/deluge
-  chown ${u}: /home/${u}/torrents/deluge
+  mkdir -p "/home/${u}/${watch_dir}"
+  chown ${u}: "/home/${u}/${watch_dir}"
+  mkdir -p "/home/${u}/$download_dir"
+  chown ${u}: "/home/${u}/${download_dir}"
```

Comment:
Should this also make and set owner for the other two directories when they're specified? will that happen automatically when the user enables them? If not, what about the watch directory, which isn't enabled by default?